ar_exercise/exercise.rb

```diff
@@ -29,20 +29,38 @@ class Apartment < ActiveRecord::Base
 
 # Hint, the following methods will help: `where`, `all`, `find`, `find_by`
 
-# Use Active record to do the following, and store the results **in a variable**
-# example: get every tenant in the DB
-all_tenants = Tenant.all
-
-# get the first tenant in the DB
-# get all tenants older than 65
-# get all apartments whose price is greater than $2300
-# get the apartment with the address "6005 Damien Corners"
-# get all tenants in that apartment
-
-# Use `each` and `puts` to:
-# Display the name and ID # of every tenant
-# Iterate over each apartment, for each apartment, display it's address and rent price
-# Iterate over each apartment, for each apartment, display it's address and all of it's tenants
+# # Use Active record to do the following, and store the results **in a variable**
+# # example: get every tenant in the DB
+# all_tenants = Tenant.all
+#
+# # get the first tenant in the DB
+# first_tenant = Tenant.first
+#
+# # get all tenants older than 65
+# older_tenants = Tenant.where("age > ?", 65)
```

Comment:
It's good to get in the habit, as you're doing, to always pass in values to AR where strings using the `?` syntax to protect against SQL injection. Happy to talk more about this if you want.